WordPress/Classes/Utility/BuildInformation/BuildConfiguration.swift

```diff
@@ -1,23 +1,18 @@
 enum BuildConfiguration: String {
-    /// Development build, usually run from Xcode
+    /// Development build, usually run from Xcode.
     case localDeveloper
 
-    /// Continuous integration builds for Automattic employees to test branches & PRs
-    case a8cBranchTest
+    /// Preproduction builds for Automattic employees.
```

Comment:
```suggestion
    /// Preproduction builds.
```

While it's currently true that the alpha build is for Automattic only, it's not a rule set in stone, right? I fee like removing make us a bit more flexible.

Reply:
Sure, we can update this later if needed. This PR already got auto-merged.